environment.yml

```
@@ -33,4 +33,5 @@ dependencies:
   - ndx-pose
   - pip
   - pip:
-    - "--editable=.[dev]"
\ No newline at end of file
+    - "--editable=.[dev]"
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)